static/js/components/PersonalForm.js

```diff
@@ -39,6 +40,28 @@ export default class PersonalForm extends ProfileFormFields {
     ui:                     UIState,
   };
 
+  renderRomanizedFields = () => {
+    const { profile } =  this.props;
+    if (showRomanizedFields(profile)) {
```

Comment:
The name for this function is a bit ambiguous: it's called `renderRomanizedFields`, but if the Romanized fields shouldn't be shown, then the function won't actually render anything at all. Could you split this up into two functions, like this?

```js
renderRomanizedFields = () => {
  return (
    <Cell col={12}>
      ...
    </Cell>
  );
}

maybeRenderRomanizedFields = () => {
  if (shouldRenderRomanizedFields(this.props.profile) {
    return renderRomanizedFields();
  } else {
    return null;
  }
}
```

Also, `showRomanizedFields` and `renderRomanizedFields` sound like they do exactly the same thing. I would also recommend renaming `showRomanizedFields` to `shouldRenderRomanizedFields`, to make the distinction clearer.